FishError/SelfReflection
# Feature/save system

Test 1
- Go to main manu and click play
- See if the level is the intended one when the player first plays the game

Test 2
- Go to any level other than the main menu and demo1 and hit play
- Bring up the pause menu and go back to the main menu
- Click start
- See if the level is the one that has just been visited

Caveat
- Because there is not a separate button for "New Game" or "Continue," the `Saves.json` file under `Assets` folder needs to be deleted upon shipping. Or, the `LastVisitedLevel` in `Saves.json` needs to be assigned the value `1`.
- Upon completing the whole game, there is not a system currently implemented to reset `LastVisitedLevel` to `1`. So, what's gonna happen is when the player tries to play a new session, they will go straight back to the last level. This is somewhat a critical issue that might need immediate attention.
